src/main/java/net/greghaines/jesque/client/AbstractClient.java

```diff
@@ -347,17 +350,27 @@ private static void doEnqueue(final Jedis jedis, final String enqueueType, final
 
     private static void doEnqueue(final Jedis jedis, final String enqueueType, final String queue, final String jobJson, final String queueKey, final String queuesKey, final boolean jobUniquenessValidation, final long future) {
         final String pushStatus;
-        final String uniquenessValidation = String.valueOf(jobUniquenessValidation);
+        LOG.info("Starting " + enqueueType + " job" + jobJson + " to the queue " + queue + " with delay " + future);
```

Comment:
This is harder to search in logz.io... please start with something more searchable.
Why aren't you using the parameters?